packages/augur-sdk/src/state/db/MarketDB.ts

```diff
@@ -270,6 +270,7 @@ export class MarketDB extends DerivedDB {
   }
 
   private processDisputeCrowdsourcerCompleted(log: ParsedLog): ParsedLog {
+    this.locks['processDisputeCrowdsourcerCompleted'] = true;
```

Comment:
@robert-e-davidson3 Where is this lock being "cleared"?

Reply:
Disregard. The diff made it look like it was clearing in the wrong place. Looking at the full file It is fine.